msg/versioned/VehicleOdometry.msg

```diff
@@ -1,34 +1,35 @@
-# Vehicle odometry data. Fits ROS REP 147 for aerial vehicles
+# Vehicle odometry data
+#
+# Fits ROS REP 147 for aerial vehicles
```

Comment:
FYI first line is short description. Space then following comment lines are long description. Improves rendering. 
Normally I'd have more description about what this is used for.